crates/zk-por-core/src/parser.rs

```diff
@@ -258,28 +258,27 @@ pub fn parse_account_state(parsed_data: &Map<String, Value>, tokens: &Vec<String
 
     let token_map = parsed_data
         .get(ASSETS_KEY)
-        .expect(format!("Account {:?} dont have key `tokens`", parsed_data).as_str())
+        .expect(format!("Account {:?} dont have key `{:?}`", parsed_data, ASSETS_KEY).as_str())
         .as_object()
         .unwrap();
     let mut parsed_equities = Vec::new();
     let mut parsed_debts = Vec::new();
 
     for token in tokens.iter() {
-        let parsed_token = token_map.get(token);
-        if parsed_token.is_none() {
-            continue;
+        let mut parsed_equity = 0 as u64;
+        let mut parsed_debt = 0 as u64;
+        if let Some(parsed_balance) = token_map.get(token) {
+            let parsed_balance = parsed_balance.as_str().unwrap().parse::<i64>().unwrap();
```

Comment:
Shouldnt reuse parse_balance variable name, maybe name it parsed_token_str

Reply:
fixed and reviewed.